mcp/mcp.c

```diff
@@ -613,7 +527,6 @@ blast_info("Finished initializing Beaglebones..."); */
 
   pthread_create(&pilot_send_worker, NULL, (void *) &pilot_compress_and_send, (void *) telemetries_linklist);
   pthread_create(&highrate_send_worker, NULL, (void *) &highrate_compress_and_send, (void *) telemetries_linklist);
-//  pthread_create(&bi0_send_worker, NULL, (void *) &biphase_writer, (void *) telemetries_linklist);
```

Comment:
@ianlowe13 why is this disabled?

Reply:
Disabled until we have a bi0 card in order to prevent unknown state of the GPIO pin from changing the status of the inCharge variable during laboratory testing